.jshintrc

```diff
@@ -11,6 +11,7 @@
   "prototypejs": false,
   "mootools": false,
   "dojo": false,
+  "esversion": 5,
```

Comment:
Eventually we may want to bump it, better to be explicit. Almost used template literals in this PR :)